homeassistant/components/binary_sensor/wink.py

```diff
@@ -117,16 +117,21 @@ def device_class(self):
         """Return the class of this sensor, from DEVICE_CLASSES."""
         return SENSOR_TYPES.get(self.capability)
 
+    @property
```

Comment:
I think this happens automatically - you don't need to define it.

Reply:
I think I tried that and got some error from the other sensor that inherit from this about super not having that property.

Reply:
Was it a lint error? In that case you can disable that.

Reply:
No it caused a stack trace. The subsensor that inherit from WinkBinarySensorDevice were blowing up on the call to super stating WinkBinarySensorDevice doesn't have a device_state_attributes property. Is it possible to call the supers super method because that is basically what I need.

Reply:
Yes, sorry, of course. Every target of the super call needs to have the method/property in question. Target is defined in the method resolution order of the class.